pyfr/solvers/euler/kernels/entropy.mako

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+<%inherit file='base'/>
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<% inf = 1e20 %>
+<%pyfr:macro name='compute_entropy' params='u, d, p, e'>
+    d = u[0];
```

Comment:
Not sure if there is a benefit to this as we're just extracting an element.

Reply:
d is used in the get_minima macro and it also helps with readability